README.md

```diff
@@ -1 +1,85 @@
-"# web-scraper" 
+# Email Scraper
+
+Extracting email addresses from websites
+
+## Intro
+
+The email scraper is a java implementaion of web crawler to scrape email addresses by crawling webs with breadth-first-search.
+It is used for scaping the email addresses from an input domain name.
+
+## Dependency and Installation
+
+* java: https://java.com/en/download/
+* gradle: https://gradle.org/install/
+* jsoup (maven dependency): https://jsoup.org/
+
+## Usage
+
+The email scraper reads a domain name from command line input and scrapes email addresses from that domain only.
+Please note that the sub-domains are explicited excluded from the web crawling.
+So the email addresses in the websites of the sub-domains will not be discovered.
+
+Command format:
+```
+java -jar path_to_your_location\email-scraper-all.jar your_domain_name
+```
+
+Example command:
+```
+java -jar build\libs\email-scraper-all.jar web.mit.edu
```

Comment:
thanks for making a pull request, including this README, and writing these clear instructions. the big thing missing here is the build command, which you did email me but could have just included here.

Reply:
also, 100 seems small to me as a default. I think pretty much any computer should be able to handle 10 or 100 times that. did you test different numbers to see how large a number you could get away with?

Reply:
Hi Dan, Appreciate your comments. All your points are very valuable. I have added the build command in README, and updated 100 to 10000 as a default. I tested 1000 and 10000 separately. It looks working fine. For 1000, it takes about 3 mins. For 10000, it takes roughly 30 mins. It looks this email scraper can take a much bigger number than 10000, just needs to run very long time. So far, I do not have an exact answer yet.